doc/sources/index.rst

```diff
@@ -49,38 +49,79 @@ Important Links
 - `About daal4py <https://github.com/intel/scikit-learn-intelex/tree/master/daal4py>`_
 
 .. toctree::
-   :caption: About
+   :caption: Getting Started
    :hidden:
+   :maxdepth: 2
```

Comment:
we are getting "how this works" item shown that make no sense - i think that maxdepth should be 1

Reply:
Fixed in https://github.com/intel/scikit-learn-intelex/pull/1323/commits/18c610980943defa287851ff154401a7f6fc9911